lib/presentation/molecules/youtube_player_molecule.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:youtube_player_flutter/youtube_player_flutter.dart';
+
+class YoutubePlayerMolecule extends StatelessWidget {
+  const YoutubePlayerMolecule({
+    required this.url,
+    super.key,
+    this.hideThumbnail = false,
+    this.showVideoProgressIndicator = true,
+    this.progressIndicatorColor = Colors.red,
+    this.playedColor = Colors.red,
+    this.handleColor = Colors.red,
+    this.padding = const EdgeInsets.all(8.0),
```

Comment:
All properties that are not being used in the contractor can get deleted, use them directly (because it is the molecule).

Reply:
please elaborate